app/controllers/users_controller.rb

```diff
@@ -8,7 +8,10 @@ def index
     @users = User.sort_by_name.page(params[:page]).per(Settings.collections.users_default_page)
   end
 
-  def show; end
+  def show
+    @user = User.find params[:id]
```

Comment:
please use method find_by instead find and don't forget handle case return nil :D